### ISSUE TITLE
cachix watch-exec prints to stdout

### ISSUE THREAD

**Issue by simonvpe** [cachix/cachix#379]
`cachix watch-exec` prints to stdout, which if you rely on parsing the output of the command you run will make things hard since the information generated by `cachix watch-exec` will be intermingled with the generated data from your command`. The solution is to write the log messages to stderr instead.

```
$ cachix watch-exec nix develop -c bats --formatter junit test > report.xml
$ cat report.xml
Watching /nix/store for new store paths ...
compressing and pushing /nix/store/... (32.3KiB)
<?xml version="1.0" encoding="UTF-8"?>
...
Stopped watching /nix/store and waiting for queue to empty ...
Done.
```

**Comment by simonvpe** [comment]
The current workaround for this problem is to run `nix develop` in two steps, like so

```
$ cachix watch-exec nix develop -c echo
$ nix develop -c bats --formatter junit test > report.xml
$ cat report.xml
<?xml version="1.0" encoding="UTF-8"?>
...
```

**Comment by domenkozar** [comment]
I'll make a release this week to fix this.

**Comment by domenkozar** [comment]
I've released Cachix 0.6.1 that fixes this!
